ydb/docs/en/core/concepts/_includes/index/how_it_works.md

```diff
@@ -1,28 +1,28 @@
 ## How it works?
 
-Fully explaining how YDB works in detail takes quite a while. Below you can review several key highlights and then continue exploring documentation to learn more.
+Fully explaining how {{ ydb-short-name }} works in detail takes quite a while. Below you can review several key highlights and then continue exploring documentation to learn more.
```

Comment:
Changes to this file look good, but they'd better be a separate PR as they are unrelated to tracing